src/core/tree.ml

```diff
@@ -279,7 +279,7 @@ module CM = struct
 
   (** [pp ?(pp_cond=false) o m] prints matrix [m] to out channel [o]. The flag
       [pp_cond] indicates whether the condition pull should be printed. *)
-  let pp : ?pp_cond:bool -> t pp = fun ?(pp_cond=false) oc m ->
+  (*let pp : ?pp_cond:bool -> t pp = fun ?(pp_cond=false) oc m ->
```

Comment:
Please, either uncomment or remove the code.

Reply:
This function is unused. @gabrielhdt do you want to keep it?

Reply:
Well, it sure is useful for debugging, but one can always reimplement it. It can be removed.

Reply:
I mean it is more up to you to decide, rather we want easily debuggable code at the expense of some dead code; or we want to avoid forgotten piece of code. I don't have a clear stand on this question.

Reply:
Well, then I am going to keep just in case.